ant-node/src/bin/antnode/main.rs

```diff
@@ -276,13 +276,14 @@ fn main() -> Result<()> {
     let (log_output_dest, log_reload_handle, _log_appender_guard) =
         init_logging(&opt, keypair.public().to_peer_id())?;
 
-    let rt = Runtime::new()?;
-    let mut bootstrap_cache = BootstrapCacheStore::new_from_peers_args(
-        &opt.peers,
-        Some(BootstrapCacheConfig::default_config()?),
-    )?;
-    // To create the file before startup if it doesn't exist.
-    bootstrap_cache.sync_and_flush_to_disk(true)?;
+    let mut bootstrap_cache = BootstrapCacheStore::new_from_peers_args(&opt.peers, None)?;
+    // If we are the first node, write initial cache to disk.
+    if opt.peers.first {
+        bootstrap_cache.write()?;
+    } else {
```

Comment:
I think if a node is running as a `service` it could restart and it would still retain the `--first` flag. 
Thus, the entire cache would be wiped.

This is only really a problem for our `genesis VM`.  But a wiped cache there is not really a big deal. If there is a better way to solve this, then great, else no issues tbh.

Reply:
I edited the RPC restart to remove the `--first` flag in [`ff94c00`](https://github.com/maidsafe/autonomi/pull/2630/commits/ff94c00af6d61719c87610dc5105c1dee738ac10)

Reply:
No I mean, a `--first` node started using antnodemanager would run as a `service`.
And services preserve their args in the service definition file.